precode.sh

```diff
@@ -6,38 +6,40 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir -p ./task ./task/dir1 ./task/dir2 ./task/dir3 ./task/dir3/dir4
 # изменяем текущую директорию на task
-
+cd ./task
```

Comment:
Во всех командах можно убрать `./`
```suggestion
cd task
```